src/components/AccordionButton/AccordionButton.module.css

```diff
@@ -0,0 +1,49 @@
+/*------------------------------------*\
+    # ACCORDION BUTTON
+\*------------------------------------*/
+
+/**
+ * Accordion Button, wraps the heading and open indicator icon.
+ */
+.accordion-button.accordion-button {
```

Comment:
This and the rest of the the `AccordionButton` styles had difficulty overriding some of the styling in [ClickableStyle](https://github.com/chanzuckerberg/edu-design-system/blob/next/src/components/ClickableStyle/ClickableStyle.module.css#L10) hence the double selectors. @dierat do you have any suggestions for avoiding the double selectors?

Reply:
Mm yeah that's tough. I guess if we want we could try using `:where`, or we oculd combine the class names so something like `.clickable-style--primary.clickable-style--brand` would become `.clickable-style--primary--brand`. For adding the classes to the component, we could change
```
      variant === 'primary' && styles['clickable-style--primary'],
      [...]
      status === 'brand' && styles['clickable-style--brand'],
``` 
to 
```
      styles['clickable-style--primary'],
      [...]
      styles[`clickable-style--${variant}--${status}`],
``` 
What do you think? Worth it? Not just for the `Accordion` but also for consumers in LP, if we want people to be able to override the `Button` and `Link` styles like this.

Reply:
The specificity issue here was with `.clickable-style` itself and not the more specific variant styling.

I want to say it wouldn't be super necessary if we want to intentionally constrain `Buttons` `Links` and other `ClickableStyle` to the options available, and with `AccordionButton` I can use a native `button` instead and apply similar styling that I would get from using the component `<Button>`.

For other cases, do you think there would be other similar case as this AccordionButton? Would we lose potential consumers if the styling was not overridable?

Reply:
>The specificity issue here was with .clickable-style itself and not the more specific variant styling.

So would we need to use `:where` to reduce that specificity to allow you to override it more easily?

>with AccordionButton I can use a native button instead and apply similar styling that I would get from using the component <Button>. (lmk your thoughts on this)

I was actually wondering this and I think it depends on how design feels about the hover state. If we want the hover state to be consistent with the other icon buttons, I think using the `Button` EDS component makes sense because that will keep them in sync. If Sean wants something else, it might be easier to use a `button` element.

Reply:
>For other cases, do you think there would be other similar cases as this AccordionButton? Would we lose potential consumers if the styling was not overridable?

Ideally it would only be cases like this where they're making a reusable thing that is different enough from the standard button that it requires unique styling, and if it is very different from our standard buttons, that also begs the question whether we should just use a native `button` instead. So I think I'm okay with people having to increase the specificity to override it since it should be pretty rare. What do you think?

Reply:
Yea I agree with that.
I'll update this button with the native `button` once I get feedback regarding the focus/hover states.

Reply:
Sean is fine with the styling here except for the border-radius, which I updated here and will update accordingly in the other Accordion pr